benchmarkapps/PlatformBenchmarks/Program.cs

```diff
@@ -3,19 +3,21 @@
 
 using System;
 using System.Net;
-using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Server.Kestrel.Core;
-using Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal;
+using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http;
 using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
 
 namespace PlatformBenchmarks
 {
     public class Program
     {
         public static void Main(string[] args)
         {
+            Console.WriteLine(BenchmarkApplication.ApplicationName);
+            Console.WriteLine(BenchmarkApplication.Paths.Plaintext);
+            Console.WriteLine(BenchmarkApplication.Paths.Json);
+            DateHeader.SyncDateTimer();
```

Comment:
Does this really make a difference?

Reply:
Does an upfront initalize of the statics in DateHeader https://github.com/aspnet/KestrelHttpServer/pull/2569#issuecomment-388659044, so its not checked every call - normally wouldn't have much impact, but its being done 7 million times a second so...
```diff
 G_M36647_IG20:
-       48B94829EC53A1020000 mov      rcx, 0x2A153EC2948
-       BA02000000           mov      edx, 2
-       E8D561825F           call     CORINFO_HELP_GETSHARED_NONGCSTATIC_BASE
-       48B94829EA43F7010000 mov      rcx, 0x1F743EA2948
+       48B94829EC53A1020000 mov      rcx, 0x2A153EC2948
        488B09               mov      rcx, gword ptr [rcx]
        4885C9               test     rcx, rcx
        7506                 jne      SHORT G_M36647_IG21
        33D2                 xor      rdx, rdx
        33C0                 xor      eax, eax
        EB07                 jmp      SHORT G_M36647_IG22
```